src/PostgREST/App.hs

```diff
@@ -372,11 +378,16 @@ treeRestrictRange maxRows_ request = pure $ nodeRestrictRange maxRows_ `fmap` re
     nodeRestrictRange :: Maybe Integer -> ReadNode -> ReadNode
     nodeRestrictRange m (q@Select {range_=r}, i) = (q{range_=restrictRange m r }, i)
 
-buildReadRequest :: Maybe Integer -> [Relation] -> [(Text, Text)] -> ApiRequest -> Either Text ReadRequest
-buildReadRequest maxRows allRels allProcs apiRequest  =
+mapLeft :: (a -> b) -> Either a c -> Either b c
```

Comment:
From the lens [docs](https://hackage.haskell.org/package/lens-4.15.1/docs/Control-Lens-Prism.html) it seems like `mapLeft foo` is the same as `& _Left %~ foo` but the lens way might be gratuitously terse.